docs/pom.xml

```diff
@@ -174,6 +174,11 @@
             <artifactId>jakarta.inject-api</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
I think I am unhappy with this being a core dependency. We kept core really slim and adding something that not a lot of folks might use is IMHO counter to that.

Reply:
The new dependency is 100% optional and the code continues to work fine without it entirely.

Reply:
Please note this is the `docs` project. The dependency is only here so the Javadoc can compile.

Reply:
d'oh.